backend/app/routers/auth.py

```
@@ -108,3 +108,32 @@
         raise_invalid_credentials()
     token = create_access_token({"sub": user.email}, timedelta(minutes=30))
     return {"access_token": token, "token_type": "bearer"}
+
+
+@router.post("/google-signup")
+def google_signup(id_token: str, db: Session = Depends(get_db)):
```

Ruff (0.8.2)

Missing return type annotation for public function `google_signup`

(ANN201)